manifests/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+kind: Kustomization
+apiVersion: kustomize.config.k8s.io/v1beta1
+commonLabels:
+  app: rhoai-edge-pipelines
+  app.kubernetes.io/part-of: rhoai-edge-pipelines
+resources:
+- tasks/
+- aiedge-e2e.pipeline.yaml
+- gitops-update-pipeline.yaml
+- https://raw.githubusercontent.com/tektoncd/catalog/main/task/github-open-pr/0.2/github-open-pr.yaml
```

Comment:
To control the version of Tekton tasks that are supported for this worklow, we should commit a copy of the versioned TektonHub tasks to this repo with some documentation about the source location and how to update it.

I would recommend creating a tasks folder dedicated to official TektonHub tasks so that it is clear that any task in this folder are sourced from the official TektonHub

Reply:
@LaVLaS are you talking about the task here we are getting from github "tektoncd/catalog/main/task/github-open-pr/0.2/github-open-pr.yaml"?

And if so if I am understanding you, you are saying lets copy this to the repo and just say where it is from and how to update it

Reply:
Apologies. My comment was for future work which can be handled in a follow-up task to support versioning of the task/pipeline featureset.  This current PR is focused on migrating the current functionality to manifest directory structure so ignore the comment above

Reply:
No problem, only think blocking this now is review and the docs. Do we want to include the readme changes in this PR or do we want to get this through to open up more work for us to do